pkg/services/team/teamimpl/store.go

```diff
@@ -536,6 +540,7 @@ func (ss *xormStore) getTeamMembers(ctx context.Context, query *team.GetTeamMemb
 		sess.Join("INNER", ss.db.GetDialect().Quote("user"),
 			fmt.Sprintf("team_member.user_id=%s.%s", ss.db.GetDialect().Quote("user"), ss.db.GetDialect().Quote("id")),
 		)
+		sess.Join("INNER", "team", "team.id=team_member.team_id")
```

Comment:
> I have tested the migration against all supported backends

Excellent, thanks!

> I wonder if, as a followup, after merging this, we want to update the existing API by exposing the assigned UID in the `POST /teams` [response](https://github.com/grafana/grafana/blob/2489e3524df33a68444cd7f76073e4faba25b1a5/pkg/api/team.go#L65) and introducing new API endpoints using the UID (and finally deprecating the existing using the sequential ID)

I was talking to @ryantxu about this yesterday, it's definitely possible to wire the uids through the system further and to either introduce new api endpoints or update existing endpoints to accept uids (since we can easily disambiguate uids from sequential ids because uids always start with a letter).

At this point I'm mostly just trying to show that we can add uids to existing models (to support the k8s project) easily and without causing any disturbance to existing code or api users, so I've kept the changes to a minimum.